guides/common/modules/con_projectserver-with-multiple-manifests.adoc

```diff
@@ -0,0 +1,40 @@
+[id="projectserver-with-multiple-manifests"]
+= {ProjectServer} with multiple manifests
+
+If you plan to have more than one Red{nbsp}Hat{nbsp}Network account, or if you want to manage systems belonging to another entity that is also a Red{nbsp}Hat{nbsp}Network account holder, then you and the other account holder can assign subscriptions, as required, to manifests.
+A customer that does not have a {Project} subscription can create a Subscription Asset Manager manifest, which can be used with {Project}, if they have other valid subscriptions.
+You can then use the multiple manifests in one {ProjectServer} to manage multiple organizations.
+
+If you must manage systems but do not have access to the subscriptions for the RPMs, you must use {RHEL} {Project} Add-On.
+For more information, see https://www.redhat.com/en/technologies/management/satellite[{Project} Add-On].
+
+The following diagram shows two Red{nbsp}Hat{nbsp}Network account holders, who want their systems to be managed by the same {Project} installation.
+In this scenario, Example Corporation 1 can allocate any subset of their 60 subscriptions, in this example they have allocated 30, to a manifest.
+This can be imported into the {Project} as a distinct Organization.
+This allows system administrators the ability to manage Example Corporation 1's systems using {Project} completely independently of Example Corporation 2's organizations (R&D, Operations, and Engineering).
+
+[[satellite_server_with_multiple_manifests_image]]
+.{ProjectServer} with multiple manifests
+image::server-multiple-manifests-satellite.png[{ProjectServer} with multiple manifests]
+
+When creating a Red{nbsp}Hat Subscription Manifest:
+
+
+* Add the subscription for {ProjectServer} to the manifest if planning a disconnected or self-registered {ProjectServer}.
+This is not necessary for a connected {ProjectServer} that is subscribed using the Subscription Manager utility on the base system.
+
+* Add subscriptions for all {SmartProxyServers} you want to create.
+
+* Add subscriptions for all Red{nbsp}Hat products you want to manage with {Project}.
+
+* Note the date when the subscriptions are due to expire and plan for their renewal before the expiry date.
+
+* Create one manifest per organization.
+You can use multiple manifests and they can be from different Red Hat subscriptions.
+
+{ProjectName} allows the use of future-dated subscriptions in the manifest.
+This enables uninterrupted access to repositories when future-dated subscriptions are added to a manifest before the expiry date of existing subscriptions.
+
+Note that the Red{nbsp}Hat Subscription Manifest can be modified and reloaded to {ProjectServer} in case of any changes in your infrastructure, or when adding more subscriptions.
```

Comment:
```suggestion
Note that the Red{nbsp}Hat subscription manifest can be modified and reloaded to {ProjectServer} in case of any changes in your infrastructure, or when adding more subscriptions.
```

Vale is right about this one: https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md#capitalization

Another suggestion: reword "reloaded" to either "updated" or "reuploaded".

Reply:
Let's fix the manifest errors then, but skip the other, more general issue with wording.